### DISCUSSION TITLE
Initial Startup Issues

### DISCUSSION THREAD

**Discussion by Joyoussi** [MRColorR/money4band#12] [Q&A] [answered]
Finished setting up the .env and running the stack, encountering these issues though: 

1. the EarnApp URL is not adding my device, "Error while linking device, device is not found"
2. http://localhost:8081/ returns a ERR_CONNECTION_REFUSED error. I don't have any other servers to my knowledge. 
3. recieving this after I start the stack: "1 error(s) decoding: * error decoding 'Command': invalid command line string" (though it still says that the stack is running fine after)

Platform: 
Edition	Windows 11 Home
Version	21H2
Installed on	‎3/‎27/‎2022
OS build	22000.1219
Experience	Windows Feature Experience Pack 1000.22000.1219.0

Docker: Docker Desktop 4.14.1 (91661)

**Comment by MRColorR** [comment]
Hello, thank for contacting me and opening a discussion. the issues 1 and 2 are a consequence of issue 3. I'll be back with a new docker-compose file to test asap.

**Comment by MRColorR** [comment]
maybe this issue could derive from a [bug](https://github.com/docker/compose/issues/9965) introduced in the docker compose 2.12.2 
which version of docker compose is installed on your system? you could check it using this command: `docker compose version` . i'm on compose v. 2.12.1 and docker desktop 14.13.1 atm and start stack option is working.
In any case, if you have not configured proxies in the env file, could you try replacing the docker-compose file with the new one from the latest release ? just download the new file using "save page as..." and replace the one already present in the project folder

then try to start the stack , let me know if this fix your issue

**Comment by MRColorR** [comment, marked as answer]
I have just upgraded to docker latest version `docker desktop 4.14.1` and `docker compose 2.12.1` the project is running without issues. To tell the truth also the old version of the docker compose file seems to work but for safety i will release the new version of the project with the updated compose to completely avoid the possible problem.
please if present remove the old containers using the stop stack option or the `docker compose down` command , download the latest release  reconfigure the env and start the stack.
If your problem still persist please try to download and run the latest release on a different machine to check if it's a specific error caused by your machine configuration.
